docs/rules/no-inline-comments.md

```diff
@@ -36,6 +36,26 @@ var bar = 5;
 //This is a comment below a line of code
 ```
 
+### ignorePattern
+
+A regular expression can be provided to make this rule ignore specific comments.
+
+Examples of **correct** code for the `ignorePattern` option:
+
+```js
+/*eslint no-inline-comments: ["error", { "ignorePattern": "webpackChunkName:\s.+" }]*/
```

Comment:
Due to recent changes in `levn`, which we use to parse config values in comments, this should be `\\s` like in string literals:

```js
/*eslint no-inline-comments: ["error", { "ignorePattern": "webpackChunkName:\\s.+" }]*/
```